README.md

```diff
@@ -1,2 +1,130 @@
+  <i>BOT algorithm</i>
+</p>
+
+- We also used Ultrasonic distance sensors, Temperature sensors and Flame sensors.
+<br/>
+
+![Bot Design](https://github.com/Ch0ubey/Fire-Extinguisher-Bot/blob/675ec79065b406595411fb3a4fc21760a32cbd39/Images_videos/Images/Fire%20Detection%20with%20Coordinates.png)
+<div align="center"><em>simulation</em></div>
+<br/>
+
+- For image processing we used camera (*as a Hardware part*). The image processing Software used is OpenCV. The data about the image is fed to OpenCV, where the image is converted to HSV and GreyScale format. Approximate Centre Coordinates of the fire are calculated. This Data is then stored in a text file for the simulation software or Audrino to use.
+<br/>
+
+![Bot Design](https://github.com/Ch0ubey/Fire-Extinguisher-Bot/blob/734ad0370c93da568aa504aa3b49e69dac5bd6af/Images%20and%20Videos/Images/Image_processing_openCV.jpg)
+<div align="center"><em>Image processing</em></div>
+<br/>
+
+## Applications
+- The main objective of the fire extinguishing bot is to search for the fire and put it off. Sensors are used to evaluate the environment in which the bot is operating and allows the bot to adjust actions based on collected data. The buzzer used will be helpful in alarming people about the danger.
+- As it is an autonomus bot which uses Controllers and image processing, so it can help us to control fire without human inputs.
+- Using this bot we can minimise delays and unavailiblity of services during emergency.
+- It can be highly useful in extinguishing fire where probability of explosion is high. For eg. Hotel kitchens, LPG/CNG gas stores, etc.
+
+## Cost Structure
+| Components   |Amount(INR)  |      Quantity      |  Cost(INR) |
+|----------    |:------:|:-------------:     |------:|
+| Ultrasonic sensors     |  75    |  7     |  525 |
+| Temperature sensors    |  75    |1            |  75 |
+| Flame sensors     |   59     |1      |  59 |
+| Camera| 300 | 1|  300|
+|Arduino UNO|500|1| 500|
+|Encoder|1500|1| 1500|
+|Dc motors|1500|1| 1500|
+|Lead screw|100|1| 100|
+|**Total Cost**| | |**4559**|
+
+
+## Limitations
+- Takes time to detect fire
+- Needs manual assistance while changing the fire extinguishers
+- Usable for household fires only
+## Future Improvement
+* Install cameras in every room
+* Link with smoke sensors in the area
+* Use custom made cylinders to avoid manual assistance
+
+</p>
+
+## Team members
+1. [Akanksha vijayvergiya](https://github.com/Akanksha-247)
+2. [Eeshan punde](https://github.com/eeshanpunde14)
+3. [Ganesh kalanidhi](https://github.com/GaneshK-RKE)
+4. [Mritunjay kumar choubey](https://github.com/Ch0ubey)
+5. [Poreddy sahith reddy](https://github.com/sahithreddyporeddy)
+6. [Srushti parbat](https://github.com/SrushtiParbat)
+## Mentors
+1. [Naman agarwalla](https://github.com/naman99-agar)
+2. [Parul chaudhary](https://github.com/Parul253)
+3. [Shubham goyal](https://github.com/shubham491981)
```

Comment:
Also capitalize the surname of each member/mentor. Make sure it's similar to the one mentioned in their Github profile.

Reply:
ok sir :'(